### PR TITLE
changes to create a separate entry point for IDHub in prebid

### PR DESCRIPTION
**Pull request by pm-manasi-moghe**

### CODE REVIEW

File: init-build.sh [hunk 1]
```diff
@@ -30,6 +31,8 @@ while getopts ":p:m:t:w:" opt; do
     ;;
     w) task="$OPTARG"
     ;;
+    x) profile="$OPTARG"
```
Comment (sureshghare) [inline init-build.sh:34]: Codacy Issue found: [profile appears unused. Verify use (or export if used externally).](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=14669)

**Comment by sureshghare** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=14669&bid=48594)

**Comment by sureshghare** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 4
           

Complexity increasing per file
==============================
- src_new/idhub.js  1
- src_new/config.idhub.js  2
- gulpfile.js  1
- src_new/util.idhub.js  9
         

Complexity decreasing per file
==============================
+ webpack.config.js  -1
         

Clones added
============
- src_new/owt.js  1
- src_new/idhub.js  1
- src_new/util.js  15
- src_new/config.idhub.js  3
- src_new/config.js  3
- src_new/util.idhub.js  16
         

Clones removed
==============
+ test/util.spec.js  -2
+ src_new/adapters/prebid.js  -3
+ src_new/controllers/idhub.js  -2
         
```


See the complete overview on [Codacy](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=14669&bid=48594)

**Comment by pm-shashank-jain** [comment]
@pm-manasi-moghe do we need this one ?

**Pull request closed without merging**